src/haz3lweb/www/style.css

```diff
@@ -237,15 +237,15 @@ svg {
 /* PAGE */
 
 #page {
-  overflow-x: auto;
-  overflow-y: auto;
   height: 100vh;
```

Comment:
Setting `#page` height to `100%` instead of `100vh` seems to fix the bottom bar on mobile and works okay on my desktop browsers. Someone with more knowledge might want to weigh in.